libsignal-service/src/account_manager.rs

```diff
@@ -59,7 +60,7 @@ const PRE_KEY_MINIMUM: u32 = 10;
 const PRE_KEY_BATCH_SIZE: u32 = 100;
 const PRE_KEY_MEDIUM_MAX_VALUE: u32 = 0xFFFFFF;
 
-impl<Service: PushService> AccountManager<Service> {
+impl<Service: PushService + MaybeSend> AccountManager<Service> {
```

Comment:
I wonder, can't we pull this in the definition of PushService itself? Declare `trait PushService: Send` with the feature, and `trait PushService` without it?

Reply:
Absolutely! Can't believe it didn't cross my mind, this way is much simpler.